### PR TITLE
test: Add 3D shear test (Fraters 2021)

### PR DESCRIPTION
**Pull request by adigitoleo**
Test inspired by Fraters 2021 Fig 5 but now with the correct angle projections. No figure by default because it's an ensemble, instead the data for both plots is saved for each switch time, plotting just requires loading the saved data and giving it to `visualisation.alignment`.

Other stuff included here:
- small fix to the pole figure plotting function for correct use of the optional `strains.scsv` files in some tests
- fix typo in number of grains in `mock.params_Fraters2021` (500 -> 5000)

### CODE REVIEW

**Comment by adigitoleo** [comment]
Will run on gadi once again to confirm that it still works and then turn off draft mode.

**Review by Patol75 — DISMISSED** [review]
Looking great, well done!

**Review by Patol75 — APPROVED** [review]

File: src/pydrex/visualisation.py [hunk 1]
```diff
@@ -174,13 +174,15 @@ def pathline_box2d(
...
       grid points in the horizontal and vertical directions) which can be set to None to
       prevent drawing velocity vectors
-    - `scale` (float, optional) — scale factor for the velocity arrows
     - `aspect` (str|float, optional) — see `matplotlib.axes.Axes.set_aspect`
     - `cmap` (Matplotlib color map, optional) — color map for `colors`
     - `cpo_vectors` (array, optional) — vectors to plot as bars at pathline locations
     - `cpo_strengths` (array, optional) — strengths used to scale the cpo bars
     - `tick_formatter` (callable, optional) — function used to format tick labels
 
+    Additional keyword arguments are passed to the `matplotlib.axex.Axes.quiver` call
```
Comment (Patol75) [inline src/pydrex/visualisation.py:183]: `axes.Axes`

**Review by Patol75 — CHANGES_REQUESTED** [review]
Small typo

**Review by Patol75 — APPROVED** [review]

**Pull request merged**